io/net/virt-net/network_virtualization.py

```diff
@@ -295,6 +295,44 @@ def test_backingdevadd(self):
             self.fail("Failed to add backing device")
         self.check_dmesg_error()
 
+    def test_disable_enable_dev(self):
+        '''
+        Test if disabling and enabling of an adapter works
+        '''
+        self.disable_enable_dev('d')
+        self.is_disabled_enabled_dev('1')
+        self.disable_enable_dev('e')
+        self.is_disabled_enabled_dev('0')
+
+    def disable_enable_dev(self, option):
+        '''
+        Disable or enable interface command
+        '''
+        cmd = "chhwres -m %s -o %s -r virtualio --rsubtype vnic -p %s -s %s" % (self.server, option, self.lpar, self.slot_num[0])
+        self.session_hmc.cmd(cmd)
```

Comment:
@cforno12  everything is fine but can you add here to make sure command run without any error. like,
```
output = self.session_hmc.cmd(cmd)
if output.exit_status != 0:
     self.fail("..msg...")
```